target/linux/mpc85xx/files/arch/powerpc/boot/dts/hiveap-370.dts

```diff
@@ -0,0 +1,294 @@
+/*
+ * Aerohive HiveAP-370 Device Tree Source
+ *
+ * Copyright (C) 2017 Chris Blake <chrisrblake93@gmail.com>
+ *
+ * This program is free software; you can redistribute  it and/or modify it
+ * under  the terms of  the GNU General  Public License as published by the
+ * Free Software Foundation;  either version 2 of the  License, or (at your
+ * option) any later version.
+ */
+
+/include/ "fsl/p1020si-pre.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+/ {
+	model = "Aerohive HiveAP-370";
+	compatible = "aerohive,hiveap-370";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+		label-mac-device = &enet0;
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+		led_status: status {
+			label = "ap370:white:status";
+			gpios = <&gpio0 11 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+			linux,default-trigger = "default-on";
+		};
+    };
+
+	chosen {
+		bootargs-override = "console=ttyS0,9600";
+	};
+
+	memory {
+		device_type = "memory";
+	};
+
+	board_lbc: lbc: localbus@ffe05000 {
+		reg = <0x0 0xffe05000 0x0 0x1000>;
+		ranges = <0x0 0x0 0x0 0xffa00000 0x100000>;
+		
+		nand@0,0 {
+					#address-cells = <0x1>;
+					#size-cells = <0x1>;
+					compatible = "fsl,elbc-fcm-nand";
+					reg = <0x0 0x0 0x4000000>;
+					bank-width = <0x2>;
+					device-width = <0x1>;
+					partition@0 {
+							reg = <0x0 0x20000>;
+							label = "0Uboot";
+					};
+					partition@20000 {
+							reg = <0x20000 0x400000>;
+							label = "Uboot";
+					};
+					partition@420000 {
+							reg = <0x420000 0x40000>;
+							label = "Uboot Env";
+					};
+					partition@460000 {
+							reg = <0x460000 0x40000>;
+							label = "Boot Info";
+					};
+					partition@4a0000 {
+							reg = <0x4a0000 0x40000>;
+							label = "Static Boot Info";
+					};
+					hwinfo: partition@4e0000 {
+							reg = <0x4e0000 0x40000>;
+							label = "Hardware Info";
+					};
+					partition@520000 {
+							reg = <0x520000 0xa00000>;
+							label = "kernel";
+					};
+					partition@f20000 {
+							reg = <0xf20000 0x5000000>;
+							label = "ramdisk";
+					};
+					partition@5f20000 {
+							reg = <0x5f20000 0x500000>;
+							label = "dtb";
+					};
+					partition@6420000 {
+							reg = <0x6420000 0x39be0000>;
+							label = "ubi";
+					};
+	        };
+	};
+
+	board_soc: soc: soc@ffe00000 {
```

Comment:
Do these dual labels work?

Reply:
the dtc compiler doesn't complain about it, but it is probably better to clean it up and only have one reference.